editor/filesystem_dock.cpp

```diff
@@ -1602,6 +1602,8 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> p_selected)
 			String fpath = path;
 			if (!fpath.ends_with("/")) {
 				fpath = fpath.get_base_dir();
+				if (!fpath.ends_with("/"))
+					fpath = fpath + "/";
 			}
 			make_script_dialog_text->config("Node", fpath + "new_script.gd", false);
```

Comment:
The proper fix would be to use `fpath.plus_file("new_script.gd")` instead of `fpath + "new_script.gd"`.

Reply:
Good. I will try to fix it. Thanks for idea.